### DISCUSSION TITLE
ms store release of webview2 does not allow multiple windows to be opened in UWP

### DISCUSSION THREAD

**Discussion by mohankumar41577** [MicrosoftEdge/WebView2Feedback#4493] [Q&A]
I'm using webview2 controls in my UWP app, before given store, I installed same build, its allowing to open new window, The problem only occurs in store release. How can I fix it?

**Comment by kosmosgames** [comment]
I have faced this issue too. Signed application installed from the store doesn't open new windows from a WebView2.
